### PR TITLE
Refine Instrument data model

### PR DESCRIPTION
**Pull request by dannygoldstein**
This PR introduces some slight refinements of the `Instrument` data model, namely:

   * The introduction of a `filters` field, that is an `Array` of `Enum`. The allowed Enum values are the filters defined by our photometry library, `sncosmo`. The `filters` of an instrument can be `[]`, indicating no filters. This would be appropriate for, e.g., a Spectrograph without any imaging capabilities. 
   * Update of the `InstrumentHandler` to this modification and proper rendering of the allowed filter values for new instruments in the api docs:

Screen Shot 2020-05-26 at 10 39 22 PM

   * Requirement that a user be a system admin to update or delete instruments. This is because follow-up requests will assume a certain schema for instruments, and we dont want anyone messing around with the capabilities of the instruments or it could affect other users' follow-up requests.

### CODE REVIEW

**Comment by dannygoldstein** [comment]
failing travis only due to a transient github issue - newest commit only introduces a comment and not any substantive modifications to the code from the previous (passing) commit

**Comment by dannygoldstein** [comment]
[ch941]

File: skyportal/tests/conftest.py [hunk 1]
```diff
@@ -137,6 +137,18 @@ def manage_users_token(super_admin_user):
...
+    role = Role.query.get('Super admin')
+    token_id = create_token(
+        permissions=[a.id for a in role.acls],
+        created_by_id=super_admin_user.id,
+        name=str(uuid.uuid4()),
+    )
+    return token_id
+
+
+
```
Comment (acrellin) [inline skyportal/tests/conftest.py:151]: One too many blank lines

**Comment by acrellin** [comment]
> @dannygoldstein `load_demo_data` will also need to be updated, as `filters` is required

Scratch that -- I see that it has a default value. My mistake. Looks good.

File: skyportal/handlers/api/instrument.py [hunk 2]
```diff
@@ -170,3 +147,45 @@ def delete(self, instrument_id):
         DBSession().commit()
 
         return self.success()
+
+
+fdoc = "List of filters on the instrument. If the instrument has no filters " \
```
Comment (stefanv) [inline skyportal/handlers/api/instrument.py:152]: Why not inline this?

**Comment by stefanv** [comment]
Thanks, @dannygoldstein !

**Pull request merged**